DataFormats/ForwardDetId/interface/MTDChannelIdentifier.h

```diff
@@ -0,0 +1,77 @@
+#ifndef DATAFORMATS_MTDCHANMNELIDENTIFIER_H
+#define DATAFORMATS_MTDCHANMNELIDENTIFIER_H
+
+#include <utility>
+#include "FWCore/Utilities/interface/GCC11Compatibility.h"
+
+
+class MTDChannelIdentifier {
+
+  public:
+
+    static const MTDChannelIdentifier& GetInstance() {
+      static MTDChannelIdentifier instance(6, 5, 12, 9);
```

Comment:
this should be static const.
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-25470/32119/llvm-analysis/report-c51df0.html#EndPath

Reply:
perhaps even ```constexpr```